tests/foreman/api/test_setting.py

```diff
@@ -108,9 +107,21 @@ def test_negative_update_hostname_with_empty_fact(self):
 
         :expectedresults: Error should be raised on setting empty value for
             hostname_facts setting
-
-        :CaseAutomation: notautomated
         """
+        hostname_facts_id = [
+            ele.id
+            for ele in entities.Setting().search(
+                query={"per_page": 200, 'search': 'name="discovery_hostname"'}
+            )
+        ][0]
+        facts = entities.Setting(id=hostname_facts_id).read()
+        original_value = facts.value
+        facts.value = ""
+        try:
+            facts = facts.update(['value'])
+            self.assertNotEqual(facts.value, "", msg="Empty string")
```

Comment:
I know it's here for consistency with other tests, but since we are switching to pytest, you might as well use `assert` here instead of unittest-provided method.